subprojects/docs/src/snippets/testing/junit-xml-results/groovy/build.gradle

```diff
@@ -26,6 +26,8 @@ testResultsDirName = "$buildDir/junit-xml"
 test {
     reports {
         junitXml {
+            omitSystemErrLog = true // defaults to false
+            omitSystemOutLog = true // defaults to false
```

Comment:
❓ In general, I think it's clearer to avoid including the negation in the property name, to avoid readers of the code having to parse double negations.

So rather than a negative property with a default of false:

```
  omitSystemErrLog = true // changes the default from false to true and omits the log
```

I think using a positve property with a default of true would be better:

```
  includeSystemErrLog = false // changes the default from true to false and omits the log
```

I think you should consider implementing this feature in this manner instead.

Reply:
I was thinking about it but was not able to find the right name for the variable, I'll implement the change

Reply:
This needs to be updated to the new name.